### ISSUE TITLE
Packaged in debian, ubuntu, and other distros?

### ISSUE THREAD

**Issue by micahflee** [dm0-/window_merge#21]
I just started using this plugin and it's excellent. Have you considered starting the process to get it officially packaged by gnu/linux distributions so that it's much easier to install, or even can be installed by default?

**Comment by dm0-** [comment]
I'm glad you find it useful.

A few requests to have the plugin packaged in Debian and Ubuntu are out there, so there is apparently some interest.  However, I don't think I should get involved with package maintenance in distributions that I won't be using regularly.  The plugin's build process should remain as close to a vanilla GNU package as possible, so packaging shouldn't be too much of a burden for any interested volunteers.

Also, there are some issues that arise with the plugin in certain use cases which I haven't found ways to correct with the current state of Pidgin and GTK2.  (GTK3 functionality may provide workarounds, but who knows when a Pidgin release will support that.)  They bother me enough that I don't like to think of this project as "ready".

I've considered requesting packager status in Fedora lately for other reasons.  Maybe they would let me submit this in some pre-alpha state.  It probably won't happen in the immediate future though; I haven't had home Internet access for the last few weeks, and even now it's flaky.  Perhaps if things stabilize in a week or two, I'll be able to work at getting sponsored.  I can update this ticket if the plugin gets approved.

**Comment by p91paul** [comment]
I think your reply will be the same if I ask you to provide a package for Arch Linux... it should be quite straightforward with AUR, but if you are not an Arch user and you don't plan to become such, probably you are not interested in this. The advantage is that you must not follow any package maintenance/approval issues, since posting PKGBUILDs to AUR is not subject to any restrictions, and there is any sort of unstable packages in there.

**Comment by dm0-** [comment]
I was under the impression that it was already available in AUR thanks to volunteer maintainers.  (I have not used it myself.)  Does this not work?

https://aur.archlinux.org/packages/pidgin-window_merge/

**Comment by p91paul** [comment]
Yes, I'm sorry, I did not search deep enough (while AUR search does not return any results for pidgin window merge, google does). Since it's available, and works, you may mention it in your README file in packaging section, also if not maintained by yourself!

**Comment by nodiscc** [comment]
A request for packaging pidgin-window-merge in Debian has been filed at http://bugs.debian.org/721265. Would be nice if someone could package/sponsor it. Thanks for this great plugin

**Comment by josch** [comment]
It seems now, more than five years after this ITP has been filed, pidgin still does not come with its own single window mode. I am a DD so I can package and upload it. But I see that this repository has seen no activity for seven years which makes me wonder if it would still make sense.

**Comment by nodiscc** [comment]
@josch I don't personnally use this plugin anymore, and the RFP was closed. And yes this is unmaintained and seems to have very few users.
